dev-requirements.txt

```diff
@@ -18,6 +18,7 @@ black==24.8.0
 bluesky==1.13.0a4
 bluesky-kafka==0.10.0
 bluesky-live==0.0.8
+bluesky-stomp @ git+https://github.com/DiamondLightSource/bluesky-stomp@4dbdb6b144b4b03243a1784f2f53dc13cbbef30e
```

Comment:
This is ad-hoc

Reply:
Did you have to make any changes to the bluesky-stomp library, or can we make a 0.1.0 on that repository?

Reply:
I think we can release the bluesky-stomp library ,I did not make any changes on it